libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/junction/JunctionProcessor.kt

```diff
@@ -36,6 +37,9 @@ internal object JunctionProcessor {
             is JunctionAction.ProcessJunctionResponse -> {
                 processResponse(action.response)
             }
+            is JunctionAction.ParseRasterToBitmap -> {
+                processRaster(action.data, action.parser)
```

Comment:
We're not parsing a raster here, it's an SVG. I think we should keep the SVG naming for clarity.

Reply:
Same everywhere else, and especially in the public API.

Reply:
Oh sorry, this is junction, not signboard, let me continue scrolling.

Reply:
Nope. Junction images are raster images in `.png` format